ts/utils/dates.ts

```diff
@@ -69,16 +69,10 @@ export function format(
 }
 
 export function isExpired(expireMonth: number, expireYear: number): boolean {
-  const today: Date = new Date();
-  const currentMonth: number = today.getMonth() + 1;
-  const currentYear: number = parseInt(
-    today.getFullYear().toString().slice(2),
-    10
-  );
-  return (
-    expireYear < currentYear ||
-    (expireYear === currentYear && expireMonth < currentMonth)
-  );
+  // We pass the first day of the month, but endOfMonth() returns a Date object
+  // representing the last day of the target month.
+  const date = endOfMonth(`${expireMonth}/01/${expireYear}`);
```

Comment:
Maybe this fixes also this bug https://www.pivotaltracker.com/story/show/176870138
Could you add some other test to check if it works both with year in 2 and 4 digits?
ex
`isExpired(3, 20)` should be the same of `isExpired(3, 2020)`